frontend/src/hooks/api/actions/useFavoriteFeaturesApi/useFavoriteFeaturesApi.ts

```diff
@@ -51,6 +58,11 @@ export const useFavoriteFeaturesApi = () => {
                     title: 'Toggle removed from favorites',
                     type: 'success',
                 });
+                trackEvent('favorite', {
```

Comment:
why do we have this event in the catch block?

Reply:
It is in try block. Hmm?

Reply:
sorry weird github formatting